front/app/modules/commercial/content_builder/admin/components/CraftComponents/Button/index.tsx

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+
+// craft
+import { useNode } from '@craftjs/core';
+
+// components
+import {
+  Radio,
+  Box,
+  Button as ButtonComponent,
+  Title,
+  Input,
+  Toggle,
+  ButtonStyles,
+} from '@citizenlab/cl2-component-library';
+
+// intl
+import messages from '../../../messages';
+import { injectIntl } from 'utils/cl-intl';
+
+type ButtonProps = {
+  text: string;
+  url: string;
+  opensInNewTab: boolean;
+  type: ButtonStyles;
+  alignment: string;
+};
+
+const Button = ({ text, url, opensInNewTab, type, alignment }: ButtonProps) => {
+  const openUrl = (url: string) => {
+    const targetValue = opensInNewTab ? '_blank' : '_self';
+    window.open(url, targetValue);
+  };
+
+  return (
+    <Box
+      display="flex"
+      justifyContent={
+        alignment === 'center'
+          ? 'center'
+          : alignment === 'left'
+          ? 'flex-start'
+          : 'flex-end'
+      }
+    >
+      <ButtonComponent
+        onClick={() => openUrl(url)}
```

Comment:
Let's use the linkTo property, instead of this. Our Button component is smart and turns into a link when `linkTo` is used, making it semantic and accessible compared to navigating via `onClick`.